File: Q1/Q1.c
//# Write a function to check if a number is even or odd.
#include<stdio.h>
void main()
{
  int num;
  printf("Enter a number of your choice: \n");
  scanf("%d" , &num);

  if(num%2==0)
  {
    printf("You have entered an even number\n");
  }

  else
  {
    printf("You have entered an odd number\n");
  }
}

File: Q1/Q1.js
//Write a C function to check if a number is even or odd.

function CheckEvenorOdd(number) {
  if (number % 2 === 0) {
    console.log(`${number} is Even`);
  } else {
    console.log(`${number} is Odd`);
  }
}

CheckEvenorOdd(5);
